metrics/deploy/pubspec.yaml

```diff
@@ -0,0 +1,16 @@
+name: deploy
+description: A sample command-line application.
+# version: 1.0.0
+# homepage: https://www.example.com
+
+environment:
+  sdk: '>=2.8.1 <3.0.0'
+
+#dependencies:
+#  path: ^1.7.0
+dependencies:
+  process_run: ^0.10.12+3
+
+dev_dependencies:
+  pedantic: ^1.9.0
```

Comment:
Should we use the [lint package](https://pub.dev/packages/lint) here for static analysis? For the metrics related projects, we use it as it is more strict than pedantic.

Reply:
sure thing - this just came with template so I will switch it over